test/elementhandle.spec.js

```diff
@@ -184,6 +184,17 @@ module.exports.addTests = function({testRunner, expect}) {
     });
   });
 
+  describe('ElementHandle.isDisplayed', function() {
+    it('should return false if element is not visible in viewport', async({page, server}) => {
+      await page.goto(server.PREFIX + '/hidden-and-visible.html');
+      await page.setViewport({ width: 1000, height: 400 });
+
+      const box = await page.$('.box');
+
+      expect(await box.isDisplayed()).toBe(false);
+    });
```

Comment:
can we also add a test that checks for the truthy value?

Reply:
Yes, I'll add this test case